src/js/components/PageHeader.js

```diff
@@ -121,7 +131,8 @@ PageHeader.propTypes = {
   iconClassName: React.PropTypes.oneOfType([
     React.PropTypes.string,
     React.PropTypes.object,
-  ])
+  ]),
+  subTitleClassName: React.PropTypes.string,
```

Comment:
I don't think it's in the master PageHeader so I'm not sure what it's still doing here...